types/jasmine/index.d.ts

```diff
@@ -836,24 +836,24 @@ declare namespace jasmine {
         /** By chaining the spy with calls.count(), will return the number of times the spy was called */
         count(): number;
         /** By chaining the spy with calls.argsFor(), will return the arguments passed to call number index */
-        argsFor(index: number): any[];
+        argsFor<T = any>(index: number): T[];
```

Comment:
All of these should probably be changed to

```
allArgs<T extends any[] = any[]>(): T;
```

So that folks can't do `allArgs<number>` by accident